backend/src/main/java/ch/puzzle/pcts/dto/calculation/CalculationInputDto.java

```diff
@@ -1,14 +1,24 @@
 package ch.puzzle.pcts.dto.calculation;
 
-import ch.puzzle.pcts.dto.member.MemberDto;
-import ch.puzzle.pcts.dto.role.RoleDto;
+import ch.puzzle.pcts.dto.calculation.degreecalculation.DegreeCalculationInputDto;
+import ch.puzzle.pcts.dto.calculation.experiencecalculation.ExperienceCalculationInputDto;
 import ch.puzzle.pcts.model.calculation.CalculationState;
 import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
 
 public record CalculationInputDto(
-        @Schema(description = "The member id associated with this calculation.", exampleClasses = MemberDto.class, example = "1", requiredMode = Schema.RequiredMode.REQUIRED, nullable = false) Long memberId,
 
-        @Schema(description = "The role id of this calculation.", exampleClasses = RoleDto.class, example = "1", requiredMode = Schema.RequiredMode.REQUIRED, nullable = false) Long roleId,
+        @Schema(description = "The member id associated with this calculation.", example = "1", nullable = false) Long memberId,
```

Comment:
why do we need nullable false?
shouldn't that be the default?
or otherwise we would need that everywhere